oci/spec_opts_unix.go

```diff
@@ -118,3 +118,10 @@ func deviceFromPath(path, permissions string) (*specs.LinuxDevice, error) {
 		GID:      &stat.Gid,
 	}, nil
 }
+
+// WithCPUCFS sets the container's Completely fair scheduling (CFS) quota and period
+func WithCPUCFS(quota int64, period uint64) SpecOpts {
+	return func(ctx context.Context, _ Client, c *containers.Container, s *Spec) error {
+		return nil
```

Comment:
Implementation seems missing

Reply:
the Linux impl. is in `spec_opts_linux.go`; because the ctr command's `run_unix.go` is compiled on non-Linux, the `With` helper had to be added with empty impl. for non-Linux

Reply:
Ah, thanks